src/config/config.json

```diff
@@ -17,6 +17,8 @@
 	"PEER_NODE_PORT": 7900,
 	"REQUEST_TIMEOUT": 5000,
 	"NUMBER_OF_NODE_REQUEST_CHUNK": 10,
+	"NODE_PEERS_REQUEST_CHUNK_SIZE": 50,
+	"CHAIN_HEIGHT_REQUEST_CHUNK_SIZE": 10,
 	"PREFERRED_NODES": ["*.symboldev.network"],
```

Comment:
Is this correct naming?

Reply:
As we discussed in the discord, here is the issue created. https://github.com/symbol/statistics-service/issues/125